tests/system/src/test/java/test/robot/javafx/embed/swing/SwingNodeBase.java

```diff
@@ -199,6 +204,12 @@ public void testAbove(boolean above) {
             robot.mouseMove(clickLoc, clickLoc);
             robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
             robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
+
+            // Emulating mouse clicks on XWayland with XTEST does not currently affect the Wayland compositor,
+            // so trying to click on the window title or window body will not bring the window to the front.
```

Comment:
This is the [same issue](https://bugs.openjdk.org/browse/JDK-8280990) we have in OpenJDK, when the emulated mouse click doesn't put one window in front of another.
After this fix and applying a fix for [the hang issue](https://bugs.openjdk.org/browse/JDK-8335468), all the `SwingNodeJDialogTest` tests are passing.